lib/utils/image_assets.dart

```diff
@@ -24,6 +25,15 @@ class ImageAssets {
   static const csi = "$_committeeLogo/CSI.png";
   static const codecell = "$_committeeLogo/codecell.png";
 
+  // company logos
+  static const _companyLogo = "$_root/company_logo";
+  static const accenture = "$_companyLogo/accenture.png";
+  static const deutsche_bank = "$_companyLogo/deutsche_bank.png";
+  static const infosys = "$_companyLogo/infosys.png";
+  static const jp_morgan = "$_companyLogo/jp_morgan.png";
+  static const oracle = "$_companyLogo/oracle.png";
+  static const tcs = "$_companyLogo/tcs.png";
```

Comment:
We dont need this...

Reply:
Aah yes, I removed earlier I forgot ki we will use Firebase